### PR TITLE
Update to latest PHP due to weekends security issue

### PR DESCRIPTION
**Pull request by cooperaj**
# Purpose

Updates PHP and other php dependencies. This doesn't affect us as it was a Windows exploit but it's better to be on the newest version/s.

Bundles in the Guzzle 7 update and necessary changes (quite a few as it happens)

## Approach

Update Dockerfiles and run composer update in api and front projects

## Checklist

* [x] I have performed a self-review of my own code
* I have added relevant logging with appropriate levels to my code
* New event_codes have been documented on the [wiki page](https://opgtransform.atlassian.net/wiki/spaces/LSML2/pages/3277881441/Understanding+the+event+logs)
* I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* I have added tests to prove my work
* I have added welsh translation tags and updated translation files
* I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* I have notified the Interaction Designer of any content changes so that appropriate screenshots/flow diagram changes can be made
* The product team have tested these changes

### CODE REVIEW

**Review by nickdavis2001 — DISMISSED** [review]

File: service-front/app/composer.json [hunk 1]
```diff
@@ -132,16 +132,16 @@
...
-        "dc-run": "@dc run --rm actor-app",
+        "dc-run": "@dc run --rm -i actor-app",
         "dc-php": "@dc-run php",
         "dc-php-debug": "@dc-php -d zend_extension=/usr/local/lib/php/extensions/no-debug-non-zts-20190902/xdebug.so",
         "psalm": "@dc-php /app/vendor/bin/psalm",
         "cs-check": "@dc-php /app/vendor/bin/phpcs",
         "cs-fix": "@dc-php /app/vendor/bin/phpcbf",
         "unit-test": "@dc-php /app/vendor/bin/phpunit --colors=always",
-        "ui-test": "@dc-php -dapc.enable_cli=1 /app/vendor/bin/behat -p acceptance --colors",
+        "ui-test": "@dc-php -dapc.enable_cli=1 /app/vendor/bin/behat -p acceptance --colors -vvv",
```
Comment (gregtyler) [inline service-front/app/composer.json:144]: Just wanted to check you meant to leave this in and it wasn't just whilst debugging the PR?

File: service-front/app/test/CommonTest/Middleware/ErrorHandling/GoneHandlerTest.php [hunk 2]
```diff
@@ -8,30 +8,34 @@
...
 use Psr\Http\Server\RequestHandlerInterface;
 
 class GoneHandlerTest extends TestCase
 {
     use ProphecyTrait;
 
-    public function testReturnsGoneResponseForGoneUris(): void
+    /**
+     * @test
+     */
```
Comment (gregtyler) [inline service-front/app/test/CommonTest/Middleware/ErrorHandling/GoneHandlerTest.php:26]: I believe you can use the `PHPUnit\Framework\Attributes\Test` attribute on these tests, which is a bit more declarative
```suggestion
    #[Test]
```

**Review by allenannom — APPROVED** [review]

File: service-front/app/test/CommonTest/Middleware/ErrorHandling/GoneHandlerTest.php (hunk 2 above)
Comment (cooperaj) [reply service-front/app/test/CommonTest/Middleware/ErrorHandling/GoneHandlerTest.php:26]: Service-front hasn't been moved to PHPUnit 10 yet. I've got a rector script ready to tackle it when I find the time to sort that.

File: service-front/app/composer.json (hunk 1 above)
Comment (cooperaj) [reply service-front/app/composer.json:144]: I think i'm the only one who runs the tests this way to be honest so figured I'd leave it in :D

**Pull request merged**